src/pages/explore/PayoutsPage/components/PayoutsList.tsx

```diff
@@ -47,9 +57,14 @@ const TimeRemaining = ({ block, latestBlock, api }: { block: number; latestBlock
 
   if (blocksLeft <= 0)
     return (
-      <Badge pill bg="primary" className="me-2 p-2">
-        Matured
-      </Badge>
+      <>
```

Comment:
It does not look really beautiful for now, I just added the button to test

A can make it look better as soon as the behavior is done 🙏 


![Screenshot 2024-12-05 at 13 03 29](https://github.com/user-attachments/assets/1071b194-3d2d-4d08-b43b-1bae80de5dcc)